frontend/src/components/UserSearch.vue

```diff
@@ -153,18 +179,26 @@ export default {
         { text: "Keycloak User ID", value: "id", class: "table-header" }
       ],
       organizations: organizations,
+      clients: [],
+      clientRoles: [ "" ],
       footerProps: { "items-per-page-options": [15] },
       userSearchInput: "",
       lastNameInput: "",
       firstNameInput: "",
       usernameInput: "",
       emailInput: "",
       organizationInput: "",
+      roleInput: "",
       searchResults: [],
       userSearchLoadingStatus: false,
       advancedSearchSelected: false
     };
   },
+  async created() {
+    // TODO error handling
```

Comment:
Is error handling still needed here? It's OK to throw exceptions for the most part, unless you have some particular error case you want to cover.